libs/community/langchain_community/chains/pebblo_retrieval/base.py

```diff
@@ -104,6 +109,11 @@ def _call(
         question = inputs[self.input_key]
         auth_context = inputs.get(self.auth_context_key, {})
         semantic_context = inputs.get(self.semantic_context_key, {})
+
+        is_valid_prompt, prompt_entities = self._check_prompt_validity(question)
+        logger.info(f"is_valid_prompt {is_valid_prompt}")
+
+        _run_manager = run_manager or CallbackManagerForChainRun.get_noop_manager()
```

Comment:
`_run_manager = run_manager or CallbackManagerForChainRun.get_noop_manager()` - this is already present on L108

Reply:
Resolved